### ISSUE TITLE
Ability to change password requirements

### ISSUE THREAD

**Issue by pieman3000** [manyfold3d/manyfold#2203]
I've just installed and I have a password I want to use that doesn't meet the requirements.

There should be a way to change or disable the default requirements.

**Comment by Floppy** [comment]
Yep, I definitely agree. I want to make it a more general password-strength thing, but didn't have the time to look into `devise-security`'s options for things like "correct horse battery staple" passwords, which currently aren't accepted.

**Comment by Floppy** [comment]
I will replace what we have with https://github.com/bitzesty/devise_zxcvbn, which does a much more flexible "password strength" check, so there won't be any more strict requirements, allowing all sorts of better passwords.
